src/pages/Signup.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+// function sConsole(event) {
+//   event.preventDefault();
+//   const data = document.getElementById('LastName');
+//   console.log(data.value);
+// }
+const formData = new FormData(document.forms[0]);
```

Comment:
I would also recommend using a class instead of getting the first object of the forms! This way if we ever end up having more than 1 form (which technically shouldn't happen bc we do good coding practices), it is clear which form is which.